### ISSUE TITLE
Add last touched by a human datetime field to all content types

### ISSUE THREAD

**Issue by edmund-dunn** [department-of-veterans-affairs/va.gov-cms#11505]
## Description
We need to add a new datetime field to all content types. It will be automagically populated by functionality attached to the save button. This will allow us to track the last time a human reviewed/touched a node. 

## Acceptance Criteria
- [ ] Add datetime field to all content types. The field will be disabled but visible on the form and view in the CMS. 
- [ ] Update the Save button functionality to automatically populate the field with the date and time the node was saved. 

field to be called: Last Saved by an Editor
Helper text: This is the last time an editor interacted with this content. This field is populated automatically when you click Save. 

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [x] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by swirtSJW** [comment]
@BlakeOrgan @EWashb  
Something doesn't feel right about this.  If this is a field that is automatically set and editors can not edit it anyway, is there any reason to surface it as a field on the node-edit form? 
![image](https://user-images.githubusercontent.com/5752113/201104020-f20a7336-dd52-4472-9e29-c636b62f6280.png)

It makes sense for the value to appear on node-view as that is how anyone can see the last time a human edited it, but it seems like extra cognitive load and noise on the editorial interface.

For some reason this issue did not use our "Enhancement"  issues template so none of the design principles show, but I think having this field show in node-edit goes against this principle
![image](https://user-images.githubusercontent.com/5752113/201106030-8399f1c3-c610-406d-bc51-9666562b5e7b.png)

**Comment by EWashb** [comment]
So I've been thinking a lot about this. You raise really good points here, Swirt.
I know they cannot interact with it, but I wonder if there is value in surfacing it since we are hoping editors will start refreshing content on a more regular cadence. Keeping this field in view may encourage them to be more proactive with 6102 compliance. However, I'm curious about @BlakeOrgan's opinion here.

**Comment by ndouglas** [comment]
We could render it as just text in the sidebar, couldn't we?

**Comment by BlakeOrgan** [comment]
I talked with @edmund-dunn about this the other day and mentioned that I thought it was weird that we were surfacing this field on the edit form when it is always going to be disabled. I do think that it's good to show this "Last edited by a human" timestamp in the node view, but I don't think that doing it in the edit flow is necessary. I like @ndouglas's idea of rendering it as text in the sidebar though because the information is still important.

@edmund-dunn mentioned that he had done it this way because that is how Dave C. requested it, but I think it may be worth presenting the text in the sidebar as an option.

**Comment by BlakeOrgan** [comment]
Talked with @edmund-dunn about this and we decided to render this as text in the right sidebar with some help text.

Last saved: {timestamp}
Help text: This is the last time this content was saved either by a human or an automated process.
Updated by: {account}

Last updated by an editor: {timestamp}
Help text: This is the last time an editor updated this content.

We also discussed the possibility of adding the ability to see who was the last human to make the update, but since that would require additional development work we decided we can iterate on that in a later sprint if we need to.
